monitoring/prober/scd/test_operations_simple.py

```diff
@@ -868,40 +868,39 @@ def test_mutate_sub2(ids, scd_api, scd_session, scd_session2):
     assert resp.status_code == 400, resp.content
     req['notify_for_operational_intents'] = True
 
-  # TODO(#386): Uncomment these tests
-  # # Attempt mutation with start time that doesn't cover Op2
-  # req['extents']['time_start'] = common.make_time(time_now + datetime.timedelta(seconds=5))
-  # resp = scd_session2.put('/subscriptions/{}'.format(Sub2), json=req)
-  # assert resp.status_code == 400, resp.content
-  # req['extents']['time_start'] = common.make_time(time_start)
-  #
-  # # Attempt mutation with end time that doesn't cover Op2
-  # req['extents']['time_end'] = common.make_time(time_now)
-  # resp = scd_session2.put('/subscriptions/{}'.format(Sub2), json=req)
-  # assert resp.status_code == 400, resp.content
-  # req['extents']['time_end'] = common.make_time(time_end)
-  #
-  # # Attempt mutation with minimum altitude that doesn't cover Op2
-  # req['extents']['altitude_lower'] = common.make_altitude(10)
-  # resp = scd_session2.put('/subscriptions/{}'.format(Sub2), json=req)
-  # assert resp.status_code == 400, resp.content
-  # req['extents']['altitude_lower'] = common.make_altitude(0)
-  #
-  # # Attempt mutation with maximum altitude that doesn't cover Op2
-  # req['extents']['altitude_upper'] = common.make_altitude(10)
-  # resp = scd_session2.put('/subscriptions/{}'.format(Sub2), json=req)
-  # assert resp.status_code == 400, resp.content
-  # req['extents']['altitude_upper'] = common.make_altitude(200)
-  #
-  # # Attempt mutation with outline that doesn't cover Op2
-  # old_lat = req['extents']['outline_circle']['center']['lat']
-  # req['extents']['outline_circle']['center']['lat'] = 45
-  # resp = scd_session2.put('/subscriptions/{}'.format(Sub2), json=req)
+  # Attempt mutation with start time that doesn't cover Op2
+
+  # req['extents']['time_start'] = scd.make_time(time_now + datetime.timedelta(seconds=5))
+  # resp = scd_session2.put('/subscriptions/{}'.format(ids(SUB2_TYPE)), json=req)
   # assert resp.status_code == 400, resp.content
```

Comment:
As per https://github.com/interuss/dss/blob/master/pkg/scd/models/subscriptions.go#L147 comment, this test case doesn't seem valid anymore.